src/scripts/modules/add_destination.js

```diff
@@ -1,46 +1,45 @@
-import { addDestination } from "../store.js";
+import { addPhoto } from "../store.js";
 let form;
 let radioBtns;
-let radioVisited;
-let radioBucketlist;
-let cityInput;
-let countryInput;
+let radioHome;
+let radioPlaces;
+let captionInput;
+let dateInput;
 let photoInput;
-let fileInput;
 let formState = {
-  city: null,
-  country: null,
+  caption: null,
+  date: null,
   photo: null,
 };
 
 function cacheDom() {
-  form = document.getElementById("add-city-form");
+  form = document.getElementById("add-photo-form");
   radioBtns = document.querySelectorAll("input[type=radio]");
-  radioVisited = document.getElementById("rd-visited");
-  radioBucketlist = document.getElementById("rd-bucketlist");
-  cityInput = document.getElementById("inputCity");
-  countryInput = document.getElementById("inputCountry");
+  radioHome = document.getElementById("rd-home");
+  radioPlaces = document.getElementById("rd-places");
+  captionInput = document.getElementById("inputCaption");
+  dateInput = document.getElementById("inputDate");
   photoInput = document.getElementById("inputImg");
 }
 
 function bindEvents() {
   form.addEventListener("submit", (event) => {
     formState.id = Math.round(Math.random() * 100000);
-    addDestination(formState);
+    addPhoto(formState);
     event.preventDefault();
     form.reset();
   });
-  radioVisited.addEventListener("click", isVisited);
-  radioBucketlist.addEventListener("click", isBucketlist);
-  cityInput.addEventListener("change", (event) => {
-    updateFormState("city", capitalize(event.target.value));
+  radioHome.addEventListener("click", isHome);
+  radioPlaces.addEventListener("click", isPlaces);
```

Comment:
The UI and UX to toggle between `home` and `places` is weird, don't you think? I don't think it's intuitive what a checkbox and checkmark do there. I think you should look for another solution to edit a photo "category" :)

Reply:
Definitely, I wanted to keep the functionality but haven't figured out yet what to replace the checkboxes with or what categories would make more sense